lib/standard/base/plugin.rb

```diff
@@ -55,6 +55,8 @@ def determine_yaml_path(desired_version)
         "ruby-3.0.yml"
       elsif desired_version < Gem::Version.new("3.2")
         "ruby-3.1.yml"
+      elsif desired_version.segments[0] == 3 && desired_version.segments[1] == 2 # exactly 3.2.x so HEAD will fall through to default.
```

Comment:
Any reason not to follow the pattern above like `desired_version < Gem::Version.new("3.3")`?

Reply:
Excellent question.  I remember thinking that doing it the same way wouldn't work, but I can see now that was entirely a brain fart.  Fixing!

Reply:
Fixed (& rebuilt the commit history)